schemas/vulnerability.yml

```diff
@@ -0,0 +1,149 @@
+
+      example: 2.0, 3.0
+
+    - name: category
+      level: extended
+      type: keyword
+      short: Category of a vulnerabilty.
+      description: >
+        The type of system or architecture that the vulnerability affects. These may be platform-specific (for example, Debian or SUSE) or general (for example, Database or Firewall). For example (ex. https://qualysguard.qualys.com/qwebhelp/fo_portal/knowledgebase/vulnerability_categories.htm)
+
+      example: AIX, Firewall, Malware, Office Application, Solaris, Windows
+
+    - name: description
+      level: extended
+      type: keyword
+      short: Description of the vulnerability.
+      description: >
+        The description of the vulnerability that provides additional context of the vulnerability. For example (ex. https://cve.mitre.org/about/faqs.html#cve_entry_descriptions_created)
+
+      example: In Keybase before 2.12.6 on macOS, the move RPC to the Helper was susceptible to time-to-check-time-to-use bugs and would also allow one user of the system (who didn't have root access) to tamper with another's installs.
+
+    - name: scanner.id
+      level: extended
+      type: keyword
+      short: ID of the vulnerability.
+      description: >
+        The identification (ID) is the number portion of a vulnerability entry. It includes a unique identification number for the vulnerability. For example (ex. https://cve.mitre.org/about/faqs.html#what_is_cve_id)
+
+      example: CVE-2019-00001, CVE-2019-00002, QID: 38739
+
+    - name: scanner.vendor
+      level: extended
+      type: keyword
+      short: Name of the scanner vendor.
+      description: >
+        The name of the vulnerability scanner vendor.
+
+      example: Tenable, Qualys
+
+    - name: severity
+      level: extended
+      type: keyword
+      short: Severity of the vulnerability.
+      description: >
+        The severity of the vulnerability can help with metrics and internal prioritization regarding remediation. For example (ex. https://nvd.nist.gov/vuln-metrics/cvss)
+
+      example: Critical, High, Medium, Low, None
+
+    - name: status
+      level: extended
+      type: keyword
+      short: Status of the vulnerability.
+      description: >
+        The current status of the vulerability on the asset:
+
+        New - The first time a vulnerability is detected by a scan the status is set to New.
+        Active - A vulnerability detected by two or more scans is set to Active.
+        Fixed - A vulnerability was verified by the most recent scan as fixed, and this vulnerability was detected by the previous scan.
+        Re-Opened - A vulnerability was reopened by the most recent scan, and this vulnerability was verified as fixed by the previous scan.
+
+        For example (ex. https://qualysguard.qualys.com/qwebhelp/fo_portal/scans/vulnerability_status.htm)
+
+      example: New, Active, Fixed, Re-Opened
+
+    - name: id
```

Comment:
I think we should rename this. If it's meant to capture the ID of the report, I would name this `vulnerability.report_id`.

To me `vulnerability.id` sounds like the identifier of one of the findings in the report.

Reply:
Done.